.github/workflows/3x.yml

```diff
@@ -29,9 +29,10 @@ jobs:
       - name: Validate Code
         run: composer code:lint
       - name: Phar Build
-        run: composer phar:build
-      - name: permissions
-        run: chmod +x ./terminus.phar
+        run: |
+          composer global require humbug/box
```

Comment:
`composer global require` is fine if we're only installing one thing, but is dangerous for two or more. To avoid future cut/copy/pasta errors, could we use `composer require` in a dedicated install directory instead? 

https://pantheon.io/blog/fixing-composer-global-command

Esp. see https://github.com/consolidation/cgr#alternative-solutions

Reply:
Updated